apps/src/publicKeyCryptography/ModuloClock.jsx

```diff
@@ -56,32 +60,35 @@ const ModuloClock = React.createClass({
     const maximumTimePerSegment = Math.min(1000, maximumDuration / 10);
```

Comment:
Read as "the clock should not take the full suggested duration if it has less than ten segments, and the longest any one segment should _ever_ take is a full second."

This is a 'feel' thing.  If you used the modulo and you're trying to animate `5 % 200` with a five-second maximum duration, you'd end up with a very small time-per-segment here - you'd hardly see it animate.  In that case I want to let the segments animate slowly, even though they only take up a fraction of the clock.

What 10 ends up meaning, then, is that when the dividend (the number of segments we need to animate) is less than 10, the total runtime may be less than the clock's suggested animation duration.  Normally time-per-segment is just `maximumDuration / dividend`. If your clock speed suggests the animation should take eight seconds but your dividend is only 2, you don't want to spend four seconds per segment - here you'd end up spending 800ms per segment instead.  Then, we put an upper limit of one second per segment on this value so that very slow total durations (say, 60 seconds) don't still result in absurdly high time-per-segment values for very low dividends.

I'm open to suggestions on a better value than 10 here; it just seemed about right, the threshold at which I want every run to start taking the same amount of time for reasonable speeds.